eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -124,20 +127,14 @@ static class AssignmentTestCase {
         List<String> expectedAssignments;
     }
 
-    private void deleteFileIfExists(String fileName) {
-        File file = new File(ApplicationProvider.getApplicationContext().getFilesDir(), fileName);
-        if (file.exists()) {
-            file.delete();
-        }
-    }
-
     private void deleteCacheFiles() {
-        deleteFileIfExists(CACHE_FILE_NAME);
+        ConfigCacheFile cacheFile = new ConfigCacheFile(ApplicationProvider.getApplicationContext(), DUMMY_API_KEY);
+        cacheFile.delete();
```

Comment:
We can just leverage the delete functionality of the `ConfigCacheFile` class, which was doing the same thing.